### ISSUE TITLE
Getting the market api without the token and cookies

### ISSUE THREAD

**Issue by Ayeven** [kookehs/bdo-marketplace#21]
https://developers.veliainn.com/ Instead of "acting" as the web browser. This act as the client. I feel like forwarding this to you since I used your code.

**Comment by qwerty32123** [comment]
what is this, is a bdo api? unnoficial or official?

**Comment by Ayeven** [comment]
It is the api that used by the game client. The endpoint is diferrent from web browser. The data response are diferrent too.

**Comment by qwerty32123** [comment]
> 
> 
> It is the api that used by the game client. The endpoint is diferrent from web browser. The data response are diferrent too.

yeah i joined discord, this is definetly the way to go if u plan on make something with market, u dont need account + u can make each request with different proxy and will be total different "person" for api. Thanks for posting.

**Comment by decyferops** [comment]
That returns a 404 tho. Is there an update on the transfer to PA and an API associated with it?
